bramkouwen/Mid-Term
# Updating index.html

## What does this PR do?
* It adds content under the first h2 tag

## How do I test this PR?
*Pull down the branch
*Open site in browser and see the newly added paragraph and images

## Screenshots
None.
